web/src/app/orbs/centerpult/page.tsx

```diff
@@ -0,0 +1,25 @@
+import OrbLayout from "../components/orb-layout/orb-layout";
+
+export default function CenterpultPage() {
+  return (
+    <OrbLayout
+      title="Centerpult"
+      description="Your piece can catapult itself into the vacant center of any four, symmetric pieces, as long as occupying the empty, center tile will complete either a plus sign or an 'x' shaped pattern."
```

Comment:
I'd add explicit info that it includes both your own and enemy pieces

Reply:
TBH - I never saw this one and I do not really know what this orb does, hence I don't know where to add your comment 
Would you please write here the entire title text so I can just copy paste it?

Reply:
`description="Your piece can catapult itself into the vacant center of any four, symmetric pieces (both your own and your opponent's), as long as occupying the empty, center tile will complete either a plus sign or an 'x' shaped pattern."`